### PR TITLE
add PDFPARSE service

### PR DESCRIPTION
**Pull request by boddumanohar**
This service is written using the CRITs'  pdfinfo_service.
https://github.com/crits/crits_services/tree/master/pdfinfo_service

### CODE REVIEW

**Comment by webstergd** [comment]
Thanks boddumanohar. I look fwd to checking this out. Do you mind cleaning up your pull a bit though? Right now it is removing the example config files.

File: src/main/scala/org/holmesprocessing/totem/actors/WorkActor.scala [hunk 1]
```diff
@@ -255,4 +255,4 @@ class WorkActor(deliverytag: Long, filename: String, hashfilename: String, downl
     case msg =>
       log.error("WorkActor: received a message I cannot match against: {}", sender(), msg)
   }
-}
+}
```
Comment (webstergd) [inline src/main/scala/org/holmesprocessing/totem/actors/WorkActor.scala]: please add a new line

File: config/totem.conf.example [hunk 2]
```diff
@@ -7,7 +7,6 @@ totem {
     download_directory = "/tmp/"
     thread_multiplier = 4
     request_timeout = 1000
-    validate_ssl_cert = true
```
Comment (webstergd) [inline config/totem.conf.example]: Please dont remove the validate_ssl_cert. Looks like a merge error

File: src/main/scala/org/holmesprocessing/totem/services/pdfparse/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Pdfparse service for Holmes-Totem
+
+## Description
+
+A simple service to extract the ROP Gadgets of a binary file.
```
Comment (webstergd) [inline src/main/scala/org/holmesprocessing/totem/services/pdfparse/README.md]: Please write a quick description of the service.

**Comment by webstergd** [comment]
Made a few quick comments. looks good to far. please make the changes and i will test it out

**Comment by Ma-Shell** [comment]
The service looks good. I have 3 small points, though:

- Please run your go-code through "go fmt" so it is formatted consistent with the other services.
- Please also provide a service.conf.example (even if the only option is "HTTPBinding")
- From playing around with it, I have never gotten any result other than
`{"Comments":0,"XREF":0,"Trailer":0,"StartXref":0,"IndirectObject":0}`
Even if I've tried it with multiple different input-pdf-files (no malicious ones though). Is this the expected result? (This does not only concern your service but the python-library you used. All my pdfs had the same results when run directly through the program without your service). Do you have any pdfs that actually produced a different result? (I dug a bit into my pdfs and saw that they indeed have Comments, XREFs, etc. Does the library only count special occurencies?)

**Comment by boddumanohar** [comment]
Now I am using updated version of the python-library. This time it parses the pdf and gives values accordingly.

**Comment by Ma-Shell** [comment]
Thank you! Looking good now. For larger pdfs it takes a while but that's again due to the library and in my opinion does not prevent this from being merged.

**Pull request merged**